login.md

```diff
@@ -1,27 +1,41 @@
 
 You will be interacting with the UNIX coding environment via the command-line. 
 
 ## What is the "Command-Line"?
-The command-line in a UNIX coding environment allows you to interact with the operating system and execute commands. From the command line, you can navigate file systems, run programs, amongst other things. Think of this as a very basic way to interact with an operating system and an alternative to a mouse/trackpad. 
+The command-line in a UNIX coding environment allows you to interact with the operating system and execute commands. From the command line, you can navigate file systems, run programs, amongst other things. Think of this as a very simplified way of interacting with an operating system or an alternative to a mouse/trackpad. 
 
 ## Logging in to our Class Server 
-Sutdents will recieve an account on the server AP, which is a server instance running on Google Cloud Platform. Think of this as signing onto a remote server where you will do all your work instead of working on and saving files to your local machine. You will access the class server using SSH, which allows a secure terminal session with our class server.
+Students will recieve an account on the class server named, AP, which is a server instance running on Google Cloud Platform. Your account allows you access to a remote server where you will do all your work and save all your files. 
 
-For those of you using macOS, we recommend using its [Terminal](https://support.apple.com/guide/terminal/welcome/mac).
+For those of you using macOS, we recommend using its [Terminal](https://support.apple.com/guide/terminal/welcome/mac). If you are using a Windows machine , we recommend using [cygwin](https://www.cygwin.com/), [PuTTY](https://www.putty.org/), [MobaX](https://mobaxterm.mobatek.net/), just to name a few. 
 
-On Windows, we recommend using [cygwin](https://www.cygwin.com/), [PuTTY](https://www.putty.org/), [MobaX](https://mobaxterm.mobatek.net/), just to name a few.
-
-Type this into the command-line to establish your connection to our server: 
+Now, to log onto our remote class server, you will need to establish use SSH (Secure SHell).Type this into the command-line to establish your connection to our server: 
 
 ```yaml
 ssh YOUR_UNI@ap.cs.columbia.edu
 ```
-You should be prompted to input your password which you should have recieved in an email. Note, it might seem like you are typing in nothing when typing in your password. 
+You should be prompted to input your password which you should have recieved in an email. Don't worry, it will seem like you are not typing anything but continue and press ENTER when done. If you followed these steps closely, you should see this message appear: 
```

Comment:
```suggestion
You should be prompted to input your password which you should have received via email. Don't worry, it will seem like you are not typing anything, but continue entering your password and press ENTER when done. If you followed these steps closely, you should see this message appear: 
```